python/plugins/processing/algs/qgis/RasterCalculator.py

```diff
@@ -77,6 +78,7 @@ def type(self):
             def clone(self):
                 return ParameterRasterCalculatorExpression(self.name(), self.description(), self.multiLine())
 
+            # TODO: remove this unused method?
```

Comment:
When I was looking at this I also couldn't find any way this was called. Also `processBeforeAddingToModeler()` is not used as far as I can tell (commented out in [ModelerParametersDialog.py#L354]( https://github.com/qgis/QGIS/blob/af98ff8ba1442aa46b3c6c144a7949ebafc4857f/python/plugins/processing/modeler/ModelerParametersDialog.py#L354)).

Reply:
seems, someone started and not finished... that's seems a part of another possible strategy for the implementation e.g. leave to adaptation to the expression instead as I did in the processAlgorithm. I preferred a more direct solution than a formally correct OO approach. But I'm open to any other approach.

Reply:
removed